db/schema.rb

```diff
@@ -231,6 +231,7 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.datetime "published_at"
+    t.bigint "status_ids", array: true
```

Comment:
I am a bit concerned about the inconsistency that we have an explicit field for `status_ids` but not for `account_ids` or `tag_ids`, which itself is not exactly consistent with statuses having `mentions` and `statuses_tags` tables for storing such relations.

Reply:
Yeah, I'm a bit unsure about this one, but I don't think we need more than an array here. Also, accounts and tags are significantly faster to fetch since we have an index on that, but not for URLs. There is kind of a precedent with reports and conversations holding arrays of toots/accounts too.